### DISCUSSION TITLE
Keycloak self-signed certificate

### DISCUSSION THREAD

**Discussion by nouhouari** [fa-fifi/keycloak_wrapper#51] [Q&A]
Hi,

I'm using keycloak with a self-signed certificated that I've deployed locally.
I can access Keycloak with this  URL __https://192.168.0.10:8443/realms/master__ and everything is working fine.

But, when I call keycloakWrapper.login(), I get the exception:

```
javax.net.ssl.SSLHandshakeException: java.security.cert.CertPathValidatorException: Trust anchor for certification path not found.
```

Is there a way to accept self-signed certificate?

**Comment by fa-fifi** [comment]
I'm not really familiar with your question since I'm a frontend dev but I guess [this](https://github.com/MaikuB/flutter_appauth/issues/176) could help.
